tests/utils/inputparams/testparams.go

```diff
@@ -153,3 +171,15 @@ func GetRandomNumber() string {
 	bytes := min + rand.Intn(max)
 	return strconv.Itoa(int(bytes))
 }
+
+// GetTestConfig - Creates one instance of TestConfig
+func GetTestConfig() *TestConfig {
+	if config == nil {
```

Comment:
@shaominchen It was one of the approaches mentioned here: http://marcio.io/2015/07/singleton-pattern-in-go/
But looks like the preferred approach is to use sync.once. 
Will make this change.